api/test/fixtures/media_type_config.py

```diff
@@ -0,0 +1,161 @@
+from dataclasses import dataclass
```

Comment:
This is just moved from `api/test/unit/conftest.py` and deduplicates the much more recent and less complete `media_type` fixture removed from `api/test/integration`.